components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -323,6 +324,13 @@ subroutine ELMFatesGlobals()
         else
            call set_fates_ctrlparms('nu_com',cval='RD')
         end if
+        
+        ! Tell fates which decomposition method is being used by the host land model
+        if (use_century_decomp) then 
+           call set_fates_ctrlparms('decomp_method',cval='CENTURY')
```

Comment:
@glemieux , there is also the CTC option in E3SM.  The choice between ctc and century has no impact on how fates operates, but it seems worth it to get it right in case this changes in the future.

Reply:
https://github.com/NGEET/fates/blob/master/main/FatesInterfaceMod.F90#L1439  Looks like it is expecting all caps "CTC"

Reply:
@rgknox is CTC option the default when running fates?  There doesn't appear to be a CTC equivalent to `use_century_decomp`.  Parsing the ELMBuildNamelist file, I think this is the case, but I wasn't entirely certain I'm reading it correctly:
https://github.com/E3SM-Project/E3SM/blob/3ac96425d48c438945049297c5bb1077566bb9fc/components/elm/bld/ELMBuildNamelist.pm#L915-L927

That said, it looks like its possible to avoid calling the `-soil_comp` option entirely, so maybe it's possible to have neither option selected?

Reply:
@thorntonpe would you be able to weigh in on the defaults for soil decomposition?  Is it possible not call `-soil_comp` option and thus not call `ctc` or `century`?